packages/kbn-guided-onboarding/src/components/landing_page/observability_link_card.tsx

```diff
@@ -0,0 +1,84 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React from 'react';
+import { i18n } from '@kbn/i18n';
+import { EuiButton, EuiFlexGroup, EuiFlexItem } from '@elastic/eui';
+import type { NavigateToAppOptions } from '@kbn/core-application-browser';
+import { UseCaseCard } from './use_case_card';
+
+interface LinkCardConstants {
+  observability: {
+    i18nTexts: {
+      title: string;
+      description: string;
+    };
+  };
+}
+
+const constants: LinkCardConstants = {
+  observability: {
+    i18nTexts: {
+      title: i18n.translate(
+        'guidedOnboardingPackage.gettingStarted.linkCard.observability.cardTitle',
+        {
+          defaultMessage: 'Observe my data',
+        }
+      ),
+      description: i18n.translate(
+        'guidedOnboardingPackage.gettingStarted.linkCard.observability.cardDescription',
+        {
+          defaultMessage:
+            'Bring together application, infrastructure, and user data into a unified solution for end-to-end observability and alerting.',
```

Comment:
```suggestion
            'Add application, infrastructure, and user data through our pre-built integrations.',
```

Reworked this text with the other UX writers to 1. introduce the idea of what an integration is, 2. shorten the text so it's closer to the other cards, 3. remove the repetition from the other cards.

Reply:
Otherwise, 👍 on the rest of the text.